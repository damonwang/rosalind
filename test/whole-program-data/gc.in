>Rosalind_1955
GTTATATGGATGTTTGTCGTAAGAAATGGATGGAGTTCCAAAATCTAAGCTTTGTGCTCA
ATAAAGGAGAACGTCGAAGATAATCTCGTGGTCACCGAACGTAGACCGATTCAGTGGGAC
TGACCCTGACTAGAAGTATAAGGGGCAAACGTATGCTAGATATCAGGTGCTGGCGTAGAA
ATCAGTGGTTGTATTCGTGTGTGGCGGATTCTTTTAATAGTTCAATCGTTAGGACACTAA
ACACTATGTGGGGGGCGCACCACGAAACTTACGTGGAGAGTCCAAGTGTCGCTAACCATC
TCCCTCAGAGGTACCATTAGGGATTGTAGCATTAGAGGTTTGTAGTCTGCGCACCTCTGC
GAGTCTATCATCTTAGCCTGGAGCTCAGGCAAGGCTCACCTTAGCCAGGATGTGCAAATC
GAAAGTGCTATCCATCAATAGTTGTGAGGCTACACTGCGGTCCCCACGAAGTCTCTTGGA
GGACTATTACGAGCATATATTTCGCCCTAGTTCCGTTCTGTGGGGGCGACAAGTAAATAC
TGACATACGTGATAAGCCAAAGTTTTCTCGCGAGTGATTCACAAACAATGCCCGCAACTA
ACATCGAACCTAAACACGGCACCACTGCTGTCGCACGCCTGACGGGGCGTTCTATTTGGA
AGTACCCTTCGGTGTAGCCTCCCGCCTGCCGGACACGCATACACCCCCTGTGCCTGGGTG
AGCGCGCCGGTGAGCTCAGTCCGTAGTATGCGAACTCGTTCATTAGGTTATCTGGCGGCT
TAAATCAGCGCTGAGTCCGCGTACAACACCGATTCCGTGTCGAACAGTGTTGTTTTTAGC
CCATTACTGCTGATTGTGCTTCCCCCGTGTACAGCGTTGTATTTCACGGGTGCACCACTC
GTACCTCAATCATTACTGAGACCGATTACGTGCTTAGGACAGTCCGCATGACTAGGCACC
CTAATCCGTTGCCCCG
>Rosalind_5200
GTACCAGCTAAGCTCTAGAACTCCAACGGTGTCTTCACTTGTAAGTACCATTATGCTTTT
CCCAACTCCCAAGCCTTGCAAAAGAGGTGAGAACTCGCAAAGATTATCATGTAGAATCGA
TGGCGCCATCAATAAACTCAACACTAAATGAAACGCTTACAGACGCACCAGGTCCACCCT
TGCCGATGCTTTTCAGCCAGCGCCATAGGTCACGGCTAACTAGTTGGTCTTCCACGGCCC
GTTGACCAACTTCGGCCCAAGGGCTATCTTACGTCACTTCCCCCAGCTCGGGGTAGAAAG
GTGCTCTGTGGTATCCGGCTACCATCCCCACAATTCGCGATTGGGTTCGACTTGTCAGCT
AGTTTACAGGCATGTAATTCTGATTCTGATAGGTGTATAGTCATACTTGTAGTCACGCAA
GCGGATCTTCCATTTCGTATCCAAGCGTAAACTGGGGCTAGTGATGAGCCACACGGCGCC
GTTAGTGGCGAAGTGAAACTATTGGAGCCAGTCGTTGTTAACTCTCCCGGGCACGGTTTA
TATCTCCATCTACCGTAATGTGCTTCGTCCTCGGGACAGTCAGACAAGCATAGGTCCTTA
TCCCAATCTTCAAAACTTTGTCGGACCGGATGCGATTAAGATATTGGCGGAGGCCTATCG
CATGGGGCTGTTGACATCGCCGAGAACCTACTACAAATGCAGGCAATTCCCTGAGAAATG
CGATCTCAGGGACAACTGAATTCGGTCGTCGACCCCACTAGAGGCTAAAGTGAGTGGCAG
ACCCTCACCCAATCCCGGAGACACGAACAGTCGCGTCAGATCCAAATTC
>Rosalind_2947
TGCTCTGCCGCCGTGAGTCCATGGTGCATTGTAATTTGATCTTTGTCATACCAGTTTCCC
GCGACAGCCCCATTCGTCGCAGCGGCATAGCGTATTTTAATTCGTCCAGATTACCACGAC
ATAAGGACACTTGGCGTGGTCCCTCTGGGCATATAGGCCCCGATGCTCAATGTTATCGAA
GTCACATTCAACCCACGCGTAACTGTGCTTCCAGAAATCGGAAGGTGGTAAAGTCAGTCG
AGTCCAGCAACATGAGCAATGCTCGAAGTTAGCCCAGGCACTGAAGGCTTCGAATACATT
CCATCTGAGTATGGCTGCCCAGAACGGATCTCGTTGTTCCGACTATAGAGGCCGTTATCC
CCTGCAGCGCGATGATTAACCCGAAGAATATCTGGATCGTACGTAAATGCCGCCAATAAT
AATGGCGTATCCTGAGCGCGATTGTAACGATTTTCGCCTGATTGTCGTGCAATTCACCGA
CACCCGTCAAGATCAAACGGGTTTGAACCTACAGTCTCAAATGAGCTGTGTGTTCGGTCG
CAGATAAGTGAATGCACATACAGCTATGGTTAAGGTGCGTCTCAGATAGGTGAGTATCCA
TCCAAGGTGGGAGACCCTATCATCCCGACGACAGGCCCTAACAAAGCAGGTGACCTTCTT
TGGCCGTGGTGCTTACGCCTGGATTAACAAAATGCTCCTTTTAGAATTCGACATACAATG
GGAGTTCTTAACATGGCAAAATTAATACTTCGAATAATCGAAACCGTATTCGACGACACG
TCATCCCGTATGCCCAGTGACTGTCGCTGCCGACTCTGATCTACCCACGAACTTGTTCAA
TGTCTCTCTTTACCCTTGGGAAAAGCGTACGCGGCCCCCAACTCCCATAGCTACTTGCGA
CTCCCCTTTGGATTGGGCCAATTACGATTTATAAAAAAACGCTGTCGGCGATTCTT
>Rosalind_9413
GGCACACTCGGACTATACCCAGCTTGACCATGCTACCAACTTGATTCAGAACCGTAAATA
GCTAGCTGACCGTGACATAGCGCCGTTGACAGTATTTTACTATTTCAACTGGATCTTTCA
TATAGGAACTTCCAGTTTCAGCTTGCCCTTGCTTATCCAGCACGGCGAGGTATTAAGCAT
GTGCCAGTCGAGACTCCGTGAGAACTAGCTCACTGGGGCATAAGCGCGTTAGAAACCCCG
CTCTCCCACGAGGCGGCAAATGTCACTCTGATATTGTCCCAGTTGCTCTCAGTTCCATTA
CCTCGTAGCACCGAGGTGGCTGGGAGATAAATCGGAATGCCCAAATGTGCATGTTCAGAG
GGAATCATACCAGTGTTAGTGCGGAATCCCCAATTGAGGTTGGCGTGGACCCCGTGACGA
GACTGAGCTGGCGCGCCACCTTTTATCGGTAGAAAGTTTTAGCCCGAATCGTTACCTTCC
TAGTTGACAGTTCATGATAACTAACGCCACGCCGTGTGTTACTGCTGCATGTCTAGGTCT
CTCCCTAAGTGATTGGGACCTACGCCTAGGCTTTTGTGGCGATTCCATTTGTACGCTACA
GAGATTGAATGCGCCGTCAAGGCATGAGATGGGACGGTGGGTCGACGCTCGTGGTAAACA
CAAACGACAAGTGTCCGATTAAATCCTCAGTCGGAGAGGATCGGTGGACCGTAGTGTAAA
CTTGGCAAACAAGGATCTGACCTACTCTCCCGAACACGCCATTGCGAAGTGTGCGGGGGA
CTGGAAGATCTCTCTGAGCGTTGAGTGCGAT
>Rosalind_8541
AAGATCAAGAAATCCAACCTTTTTGCGCGCCTTGGCACAGGTAAAGCCTTTGAATGACCT
GTCAGCGCCGCGTTCTCCCATTGGAACGATCCAGCAAGGCTCAAATGCGTCGAGTTCCAA
AGCACGGTGCGCACTCTTCGTATACTCGCACGATGTTTAAAGAGGGGTGAATGAGTGCCG
ACGTGACGACATAAATGCTAGGTAAAGAAGCTACTTCTTAAATGGGTGCTGTCCAGGATC
AGCTAACCACTTCTTTGACTACGGTCTAATCGATCCGTGTCATACCACCAAGCTAGCGCG
TATCCGATGAGGGTGAAAGCACGACACGTGGGAAATGTATCCTGATGTCTTTTTCCGAAC
GGCAGTTGTCGTACACGCCGAATTTTACCACATGAAATTAGGTCGTTCGCCCATCGGGTG
ACACGCGCGCGCGCCACTTCGAGTCGAACCAAAACTTTAGTGGGGGGGGATTTTCAGCTC
GTTATTACGCACAGATTCGTTGAGATGAGTGCGGTGCTCTGACAATAAGGTAAGGGGACT
GAGACACCCAAGGACCCAGTATTCTTGCTTACCACCCTTAACGTAAAGGTGCCCCAATTT
CTATTGCTGCATTGTTTGTTTTATATTGTCCGCAACTCGCCTGCAGACCGCTGCGCTCCG
CTAAGGAGGGCGTTCCTAGTGAGTGGTATGCTACCGCCGCAGCCAAGTCGCGGCGGGGCC
TTTATGGGACCGGCGAAGACTTTAGATGGTCCCGGCCGCTGGGGTTTGGGAAGAAATGGT
TTCATCCCTGGTAGAGTGTTTTGCGGTTGACGACTCTTCCAACTTTATCTGCGGTTATAG
GCTACTCGAGAGGTGAGTACGGAAGACTTGGGTGGTTTGGCACTGTATAGGTTTGAGCGA
CGGCTTCTCGACTCTATCGCTCCCCGGCTTAACCACGCTTTAT
