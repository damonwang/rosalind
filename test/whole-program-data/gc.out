Rosalind_8541
51.537646%
